package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "app",
```

```suggestion
  "name": "rails_api_base",
```

so it can be replaced by the script?